x-pack/plugins/cases/common/api/user.ts

```diff
@@ -7,11 +7,14 @@
 
 import * as rt from 'io-ts';
 
-export const UserRT = rt.type({
-  email: rt.union([rt.undefined, rt.null, rt.string]),
-  full_name: rt.union([rt.undefined, rt.null, rt.string]),
-  username: rt.union([rt.undefined, rt.null, rt.string]),
-});
+export const UserRT = rt.intersection([
+  rt.type({
+    email: rt.union([rt.undefined, rt.null, rt.string]),
+    full_name: rt.union([rt.undefined, rt.null, rt.string]),
+    username: rt.union([rt.undefined, rt.null, rt.string]),
+  }),
+  rt.partial({ profile_uid: rt.string }),
```

Comment:
What is the difference between`rt.partial({ profile_uid: rt.string })` and `profile_uid: rt.union([rt.undefined, rt.null, rt.string])` ?

Reply:
By using a `partial` we don't have to set the `profile_uid` wherever a user object is defined. That way we can avoid updating all the locations in the code where we create a user object. I believe we can also avoid writing a migration.

Reply:
Make sense. Thanks!